src/spike_source_cell_group.hpp

```diff
@@ -6,6 +6,10 @@
 
 namespace arb {
 
+struct spike_source_cell {
+    time_seq seq;
+};
+
 class spike_source_cell_group: public cell_group {
```

Comment:
I can postpone this to the installable-target work, but it would be good to have the public-facing class (`spike_source_cell`) in a separate header from the library implementation class (`spike_source_cell_group`).